src/Tribe/Main.php

```diff
@@ -1594,9 +1594,7 @@ public function saved_organizers_dropdown( $current = null, $name = 'organizer[O
 		 */
 		public function admin_body_class( $classes ) {
 			global $current_screen;
-			if ( isset( $current_screen->post_type ) &&
-				 ( $current_screen->post_type == self::POSTTYPE || $current_screen->id == 'settings_page_tribe-settings' )
-			) {
+			if ( isset( $current_screen->post_type ) && ( $current_screen->post_type == self::POSTTYPE || $current_screen->id == 'settings_page_tribe-settings' ) ) {
```

Comment:
Also note the reversal of the scalar comparisons. Since we standardize on WP PHP standards, we might as well fix scalar conditionals as we encounter them to match [Yoda conditions](https://make.wordpress.org/core/handbook/coding-standards/php/#yoda-conditions).